### PR TITLE
Discover integration tests.

### PR DESCRIPTION
**Pull request by stevenbronson-wk**
#### Reason for change
Integration tests introduced in PRs don't run until merged.

#### Steps to Test
You could try adding a PR on a fork after incorporating this change.

**review**:
@Arelle/arelle

### CODE REVIEW

File: .github/workflows/conformance-suites.yml [hunk 1]
```diff
@@ -2,18 +2,29 @@ name: Run Xbrl Conformance Suites
...
+          find tests/integration_tests -name 'test_*.py' -not -name test_efm_conformance_suite.py | jq -Rsc 'split("\n") | map(select(length > 0))'
+
   run-conformance-suite:
+    needs: find-tests
     permissions: read-all
     runs-on: ubuntu-latest
     continue-on-error: true
     strategy:
       matrix:
-        test-path:
```
Comment (sagesmith-wf) [inline .github/workflows/conformance-suites.yml:11]: This doesn't really seem like an objective improvement, would suggest not doing this, but will leave it up to the team. My concerns are when we want to change, you now have to know `find`, and `jq`. I will leave up to the team though

Comment (aaroncameron-wk) [reply .github/workflows/conformance-suites.yml:11]: It is more complicated, but the value it adds (running your PR's new tests before merging) is worth it in my opinion.

**Review by aaroncameron-wk — APPROVED** [review]

File: .github/workflows/conformance-suites.yml (hunk 1 above)
Comment (austinmatherne-wk) [reply .github/workflows/conformance-suites.yml:11]: I'm personally fine with both `find` and `jq`. They're both pretty standard utils

**Review by austinmatherne-wk — APPROVED** [review]

**Comment by aaroncameron-wk** [comment]
QA +10 
Confirmed that tests are retrieved and run from head branch: https://github.com/aaroncameron-wk/arelle-public/pull/20

**Pull request merged**